README.md

```diff
@@ -118,6 +119,12 @@ Here is the list of supported file formats:
 * **.gltf/.glb** : GL Transmission Format (full scene)
 * **.3ds** : Autodesk 3D Studio file format (full scene)
 * **.wrl** : VRML file format (full scene)
+* **.fbx** : Autodesk Filmbox (full scene)
+* **.dae** : COLLADA (full scene)
+* **.off** : Object File Format (full scene)
+* **.dxf** : Drawing Exchange Format (full scene)
+
+> WARNING: The support of DXF files rely on Assimp library and is very limited.
```

Comment:
Add a more detailed section about assimp related limitations in the limtations section

Reply:
With the details we know, eg:

- PBR not supported for FBX
- All animations not supported for all assimp formats
- Armatures are not read
- Cameras only file may show artifacts
- Certain files can be empty
- Certain files may crash in assimp
- Animation is not supported by assimp 5.1, but supported by assimp 5.0